### PR TITLE
Amendment to ADR 003: Namespaced keywords

### PR DESCRIPTION
**Pull request by opqdonut**

### CODE REVIEW

File: docs/architecture/003-namespaced-keywords.md [hunk 1]
```diff
@@ -53,3 +56,29 @@ Pros of namespaced keys:
 Cons:
 
 - verbose (mitigated by using short namespaces)
+
+# Amendment to decision
```
Comment (Macroz) [inline docs/architecture/003-namespaced-keywords.md]: I would name this amendment specifically and include a date on it too for extra clarity. Let's say `Amendment 6th of March 2019: Same data for same key`

Comment (opqdonut) [reply docs/architecture/003-namespaced-keywords.md]: done

**Comment by opqdonut** [comment]
I think the text flows better if I show the negative examples first. But maybe they need to be tagged better.

File: docs/architecture/003-namespaced-keywords.md [hunk 2]
```diff
@@ -53,3 +56,42 @@ Pros of namespaced keys:
...
+```
+
+Instead use partial maps like this:
+
+```clojure
+{:blacklist/user {:userid "bob"}
+ :blacklist/resource {:resource/ext-id "urn:123"}}
+```
+
+Or reuse known keys like this:
```
Comment (okahilak) [inline docs/architecture/003-namespaced-keywords.md]: I didn't understand straight away what this means. Could it be clarified a bit?

File: docs/architecture/003-namespaced-keywords.md [hunk 3]
```diff
@@ -53,3 +56,42 @@ Pros of namespaced keys:
...
+
+Date: 2019-11-05
+Authors: @opqdonut @Macroz
+
+When using namespaced keys, it's useful to keep the structure of the
+corresponding values fixed. That is, don't mix forms like this:
+
+```clojure
+{:blacklist/user "bob"
+ :blacklist/resource :123"}
```
Comment (okahilak) [inline docs/architecture/003-namespaced-keywords.md]: typo: missing "

File: docs/architecture/003-namespaced-keywords.md (hunk 2 above)
Comment (opqdonut) [reply docs/architecture/003-namespaced-keywords.md]: Tried to clarify

**Pull request merged**